ui/js/controllers/perf/compare.js

```diff
@@ -58,6 +59,66 @@ perf.controller('CompareChooserCtrl', [
                 $scope.newRevision = tip;
             };
 
+            $scope.getPreviousRevision = function() {
+                $scope.proposalRevision = $scope.newRevisionError = null;
+
+                // only check for a full revision
+                if ($scope.newRevision.length != 12) return;
+
+                // indicate that we are loading if it takes some time
+                var timeout = setTimeout(function() {
+                    $scope.proposalRevisionLoading = true;
+                }, 400);
+
+                var promise;
+                if ($scope.newProject.name == "try") {
+                    // try require some special logic
+                    var iProjs = _.filter($scope.projects, function(proj) {
+                        return proj.name == "mozilla-inbound" ||
+                            proj.name == "mozilla-central" ||
+                            proj.name == "fx-team";
```

Comment:
b2ginbound is just fine- although I do wonder if that branch will exist
much longer with the tier3 status.

On Fri, Jan 29, 2016 at 9:58 AM, Julien Pagès <notifications@github.com>
wrote:

> In ui/js/controllers/perf/compare.js
> <https://github.com/mozilla/treeherder/pull/1283#discussion_r51292488>:
>
> > +
> > +                // only check for a full revision
> > +                if ($scope.newRevision.length != 12) return;
> > +
> > +                // indicate that we are loading if it takes some time
> > +                var timeout = setTimeout(function() {
> > +                    $scope.proposalRevisionLoading = true;
> > +                }, 400);
> > +
> > +                var promise;
> > +                if ($scope.newProject.name == "try") {
> > +                    // try require some special logic
> > +                    var iProjs = _.filter($scope.projects, function(proj) {
> > +                        return proj.name == "mozilla-inbound" ||
> > +                            proj.name == "mozilla-central" ||
> > +                            proj.name == "fx-team";
>
> @wlach <https://github.com/wlach> Oh yeah, this is quite what we wanted
> (or maybe better actually). In there we have
>
> ["mozilla-inbound", "mozilla-central", "fx-team", "b2g-inbound"]
>
> So basically, b2g-inbound is added in comparison to my patch. @jmaher
> <https://github.com/jmaher> is that ok if we add that branch ?
>
> Also, it is used here:
>
> ui/js/controllers/perf/graphs.js:809:                                            testsDisplayed, options, thPerformanceBranches) {
> ui/js/controllers/perf/graphs.js:907:        thPerformanceBranches.forEach(function (branch) {
>
> The meaning of this value should be the same everywhere (obviously), but
> I'm not qualified to judge that. Is is good ? Else I could create a new
> *value* for this specific need.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/treeherder/pull/1283/files#r51292488>.
>